2-Browsers/Week1/homework/ex5-catWalk/index.js

```diff
@@ -15,8 +16,50 @@
    for 5 seconds, and then replace the img with the original image and have it 
    continue the walk.
 -----------------------------------------------------------------------------*/
+//crating img and changing style of img
+const img = document.getElementsByTagName('img')[0]; // [0] i used it for first desired image, because it returns an array and i want to take first image
+img.style.left = '0px';
+
+const dancingCat = 'https://media1.tenor.com/images/2de63e950fb254920054f9bd081e8157/tenor.gif?itemid=10561424'
+const originalImgSrc = img.src;
+const originalImgWidth = img.width;
+let walkForwards = true;
+
+//moving cat to right 10px
 function catWalk() {
-  // TODO complete this function
+
+  let currentLeft = parseInt(img.style.left); //converting px to int number
```

Comment:
`currentLeft` doesn't change so this can be a `const`.

To prevent any unexpected situations you should always use the second parameter of parseInt: `parseInt(..., 10)`